src/SpectodaRuntime.ts

```
@@ -495,17 +502,17 @@ export class SpectodaRuntime {
    */
 
-  addEventListener(event: SpectodaTypes.JsEvent, callback: Function) {
+  addEventListener(event: SpectodaJsEventName, callback: Function) {
     return this.on(event, callback);
   }
   /**
    * @alias this.addEventListener
    */
-  on(event: SpectodaTypes.JsEvent, callback: Function) {
+  on(event: SpectodaJsEventName, callback: Function) {
     return this.#eventEmitter.on(event, callback);
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)